apps/wallet/src/ui/app/shared/ButtonUI.tsx

```diff
@@ -129,6 +129,7 @@ export const Button = forwardRef(
             <ButtonOrLink
                 ref={ref}
                 className={styles({ variant, size })}
+                disabled={disabled}
```

Comment:
good catch, (wondering how it works now?) but we could stop destructuring disabled and let it be in the otherProps instead since it seems we not use it anywhere else